src/components/Modals/NewCourse/SelectedRequirementEditor.vue

```diff
@@ -1,75 +1,52 @@
           {{ requirementsThatAllowDoubleCounting.join(', ') }}
         </div>
       </div>
     </div>
-    <div v-if="relatedRequirements.length > 0">
+    <div v-else class="newCourse-requirements-container newCourse-requirements">
+      This class does not automatically fulfill any requirements.
+    </div>
+    <div v-if="nonAutoRequirements.length > 0">
       <div class="newCourse-title">This class can fulfill the following requirement(s):</div>
-      <div v-if="!editMode" class="newCourse-requirements-container">
-        <div class="newCourse-requirements">
-          {{ relatedRequirements.map(it => it.name).join(', ') }}
+      <div v-if="!editMode">
+        <div class="newCourse-requirements-container">
+          <!-- eslint-disable-next-line vue/no-v-html  -->
+          <div class="newCourse-title" v-html="nonAutoRequirementsText"></div>
         </div>
       </div>
-      <div v-else class="newCourse-requirements-edit">
-        <edit-single-requirement
-          v-for="relatedRequirement in relatedRequirements"
-          :key="relatedRequirement.id"
-          :name="relatedRequirement.name"
-          :selected="selectedRequirementID === relatedRequirement.id"
-          :isClickable="true"
-          @on-select="() => toggleSelectRequirement(relatedRequirement.id)"
-        />
-      </div>
-    </div>
-    <div v-if="potentialRequirements.length > 0">
-      <div class="newCourse-title">
-        This class could potentially fulfill the following requirement(s):
-      </div>
-      <div v-if="!editMode" class="newCourse-requirements-container">
-        <div class="newCourse-name">
-          {{ potentialRequirements.map(it => it.name).join(', ') }}
+      <div v-else>
+        <div v-if="potentialRequirements.length > 0" class="warning">
+          <img class="warning-icon" src="@/assets/images/warning.svg" alt="warning-icon" />
+          We cannot accurately check the requirements marked with the warning icon, so double check
+          before selecting.
         </div>
-      </div>
-      <div v-else class="newCourse-requirements-edit">
-        <edit-single-requirement
-          v-for="potentialRequirement in potentialRequirements"
-          :key="potentialRequirement.id"
-          :name="potentialRequirement.name"
-          :selected="selectedRequirementID === potentialRequirement.id"
-          :isClickable="true"
-          @on-select="() => toggleSelectRequirement(potentialRequirement.id)"
+        <requirements-dropdown
+          :relatedRequirements="relatedRequirements"
+          :potentialRequirements="potentialRequirements"
+          :selectedID="selectedRequirementID"
+          @on-selected-change="toggleSelectRequirement"
         />
       </div>
     </div>
-    <div v-if="!editMode" class="newCourse-link" @click="toggleEditMode()">
-      {{ editRequirementsText }}
-    </div>
+    <div v-if="!editMode" class="newCourse-link" @click="toggleEditMode()">Edit Requirements</div>
```

Comment:
Can we make this a button? The Add Modal is actually almost all keyboard accessible 🥰 except for this part!